ui.tests/test-module/specs/prefill/repeatableprefillwithzerooccurrencefortabaccordionwizard.spec.js

```diff
@@ -0,0 +1,39 @@
+/*******************************************************************************
+ * Copyright 2023 Adobe
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+describe("Form with Wizard/Tab/Accordion and Prefill with Zero occurance", () => {
+    const pagePath = "content/forms/af/core-components-it/samples/prefill/repeatableprefillwithzerooccurrencefortabaccordionwizard.html";
+    let formContainer = null;
+
+    beforeEach(() => {
+        cy.previewForm(pagePath).then(p => {
+            formContainer = p;
+        })
+    })
+
+    it("test that repeatable instances with zero occurance is not in the dom", () => {
+        const children = formContainer._model._children;
```

Comment:
Why are we not making use of public API's ? Ideally, test should make use of public API's.

Reply:
used getModel()